ninja-maven-plugin/src/test/java/ninja/build/WatchAndRestartMachineTest.java

```diff
@@ -1,88 +1,74 @@
 /**
- * Copyright (C) 2012-2017 the original author or authors.
- *
```

Comment:
Can you please undo all the changes that are not related to the two lines you changed? Otherwise we are having a hard time figuring out what happened later... Thanks!